CONTRIBUTING.md

```diff
@@ -71,13 +71,13 @@ which will allow you to run the full suite of tests. If you choose to install
 WSL, follow the Linux/Mac OS instructions above.
 
 If you do not wish to install WSL, run the following commands from a Windows
-terminal to install all non-pytype requirements:
+terminal to install all non-pytype requirements (powershell example):
 
-```
-> python -m venv .venv
-> ".venv/scripts/activate"
-(.venv) > pip install -U pip
-(.venv) > pip install -r requirements-tests.txt
+```powershell
+PS > python -m venv .venv
+PS > & ".venv/scripts/activate"
+(.venv) PS > pip install -U pip
+(.venv) PS > pip install -r requirements-tests.txt
```

Comment:
I use `terminal.exe` and have never used powershell 😶

Reply:
Out of curiosity, I don't think I've heard of `terminal.exe` (unless you mean `cmd.exe` or the [Windows Terminal](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701)
I know of a few coworkers who like using the [Cmder console emulator](https://cmder.app/)

Also I could totally remove the `&` by removing the quotes and flipping the slashes. Making it agnostic between powershell or cmd since the path has no spaces. `> .venv\scripts\activate`

Reply:
Sorry, Windows Terminal was the one I meant, yeah!

> Also I could totally remove the & by removing the quotes and flipping the slashes. Making it agnostic between powershell or cmd since the path has no spaces. > .venv\scripts\activate

SGTM

Reply:
> I know of a few coworkers who like using the [Cmder console emulator](https://cmder.app/)

Oh that one looks niiice. Ugh I might have to change.